src/tools/data_handlers/analytics_handler.py

```diff
@@ -380,10 +399,29 @@ async def pull_data(
                     )
 
             if "status" in result and result["status"] in ["success", "saved"]:
-                raw_data, data_points_count, message_detail = (
-                    self._process_response_data(result)
-                )
-
+                (
+                    raw_data,
+                    data_points_count,
+                    message_detail,
+                ) = await self._process_response_data(result)
+                logger.debug(f"RAW_DATA: {raw_data}")
```

Comment:
Seems like a very long log. Maybe truncate this

```suggestion
                logger.debug(f"RAW_DATA: {raw_data}"[:300])
```